JASP-Engine/JASP/R/LSbinomialcommon.R

```diff
@@ -594,20 +594,21 @@
 }
 .marginalCustomBinomialLS   <- function(density, spikes, lCI, uCI, densityDiscrete = FALSE){
   
-  if (!is.null(density))
-    if (!densityDiscrete)density$y <- density$y/nrow(density)    
-    else
-      density <- data.frame("y" = NULL, "x" = NULL)
-    
-    if (length(spikes) != 0){
-      for(i in 1:length(spikes)){
-        density <- rbind(density[density$x <= spikes[[i]]$x,], spikes[[i]], density[spikes[[i]]$x < density$x,])
-      }
+  if (!is.null(density)){
+    if (!densityDiscrete)
+      density$y <- density$y/nrow(density)    
+  }else
```

Comment:
This is still a bit iffy, I believe you it works now but can you simplify the logic a little to remove these nested if statemenets? I won't beat you on this but it's definitely not pretty...

Reply:
yeah, I think that it's a remnant from a time when the densityDiscrete was a part the density object itself. I will change it.